src/vendor_upstream/dom/ReactWheelHandler.js

```diff
@@ -64,6 +64,8 @@ class ReactWheelHandler {
   }
 
   onWheel(/*object*/ event) {
+    event.preventDefault();
```

Comment:
Will this lead to the issue where you can't scroll the parent page with your pointer over the grid?

Reply:
Yep, same as above.